config/manager.go

```diff
@@ -140,6 +140,11 @@ func (f *FilesystemConfigManager) save(c Config) error {
 		return err
 	}
 
+	err = f.fsHandle.CreateFile(f.location, internal.PermUserRW)
+	if err != nil {
+		return err
+	}
+
 	return f.fsHandle.WriteFile(f.location, encrypted, internal.PermUserRW)
```

Comment:
Shouldn't `WriteFile` be changed to create the folders parents?

Reply:
Maybe

Reply:
@bartoszWojciechO What do you think?

Reply:
I have only implemented the abstraction so that filesystem access can be mocked. Both ways are fine by me.